css/style.css

```diff
@@ -0,0 +1,199 @@
+.main{
+    display: flex;
+    flex-direction: column;
+    max-width: 1280px;
+    margin: 0 auto;
+    padding: 0 20px;
+    position: relative;
+}
+
+#header{
+    display: flex;
+    max-width: 100%;
+    flex-direction: row;
+    align-items: center;
+    padding: 15px, 20px;
+    position: fixed;
+    left: 5%;
+    right: 5%;
+    top: 0;
+   
+}
+
+.logo{
+    display: flex;
+    
+}
```

Comment:
There is way too much space in between the lines of the CSS codes. you should install Linters as soon as possible to avoid this